tools/ci-ts/tests/flux-monitor.test.ts

```diff
@@ -111,6 +112,12 @@ beforeAll(async () => {
   linkToken = await linkTokenFactory.deploy()
   await linkToken.deployed()
 
+  // Set up a recurring tx send in the background to ensure that the
+  // ETH node is continually mining blocks despite being in dev mode.
+  const miner = ethers.Wallet.createRandom().connect(provider)
+  await t.fundAddress(miner.address)
+  txInterval = t.setRecurringTx(miner)
```

Comment:
Is this recurring transaction still helpful to leave in here? Seems possibly debugging related and left over.

Reply:
@RyanRHall mentioned that our `devnet` image doesn't (or at least, didn't) mine blocks unless it receives transactions.  I can try the test again with this removed if you want.